tests/integration/models/test_inbox.py

```diff
@@ -90,6 +90,44 @@ def test_message(self):
         assert message.replies == []
         assert isinstance(message.subreddit, Subreddit)
 
+    @mock.patch('time.sleep', return_value=None)
+    def test_message_collapse(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+                'TestInbox.test_message_collapse',
+                match_requests_on=['uri', 'method', 'body']):
+            self.reddit.inbox.collapse(list(self.reddit.inbox.messages()))
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_message_uncollapse(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+                'TestInbox.test_message_uncollapse',
+                match_requests_on=['uri', 'method', 'body']):
+            self.reddit.inbox.uncollapse(list(self.reddit.inbox.messages()))
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_message_collapse__with_id(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+                'TestInbox.test_message_collapse__with_id',
+                match_requests_on=['uri', 'method', 'body']):
+            messages = []
+            for item in self.reddit.inbox.messages():
+                messages.append(item.fullname)
+            self.reddit.inbox.collapse(messages)
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_message_uncollapse__with_id(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+                'TestInbox.test_message_uncollapse__with_id',
+                match_requests_on=['uri', 'method', 'body']):
+            messages = []
+            for item in self.reddit.inbox.messages():
+                messages.append(item.fullname)
+            self.reddit.inbox.uncollapse(messages)
+
     def test_message__unauthorized(self):
```

Comment:
I think this method should come before all these new methods as the second `_` is less than both `c` and `u`.